test/test_solve_powerflow.jl

```diff
@@ -601,72 +488,68 @@ end
 
     @test all(
         isapprox.(
-            res_klu["bus_results"][!, :Vm],
-            res_nlsolve["bus_results"][!, :Vm],
+            res_lu["bus_results"][!, :Vm],
+            res_newton["bus_results"][!, :Vm],
             rtol = 0,
-            atol = 1e-8,
+            atol = 1e-12,
         ),
     )
     @test all(
         isapprox.(
-            res_klu["bus_results"][!, :θ],
-            res_nlsolve["bus_results"][!, :θ],
+            res_lu["bus_results"][!, :θ],
+            res_newton["bus_results"][!, :θ],
             rtol = 0,
-            atol = 1e-8,
+            atol = 1e-12,
         ),
     )
+end
 
-    # test against legacy implementation
-    pf_legacy = ACPowerFlow(PowerFlows.LUACPowerFlow)
-    res_legacy = solve_powerflow(pf_legacy, sys)
+@testset "Test loss factors for larger grid" begin
+    if Sys.iswindows()
+        return
```

Comment:
At the very least let's throw a warning here, not just silently pass. I kind of feel like we should get to the bottom of this at some point, it's pretty troubling… open an issue?